app/assets/stylesheets/styles/game.scss

```diff
@@ -6,6 +6,35 @@
 // #chessboard tr:nth-child(2n) td:nth-child(2n+1) { background: #9f9f9f; }
 // #chessboard tr:nth-child(2n+1) td:nth-child(2n) { background: #9f9f9f; }*/
 
+#cbwithgys {
+  display: flex;
+  margin-left: auto;
```

Comment:
Properties should be ordered display, justify-content, margin-left, margin-right